### ISSUE TITLE
Clean Boutiques examples

### ISSUE THREAD

**Issue by glatard** [boutiques/boutiques#142]
Update them to version 0.5
Add them to tests

**Comment by glatard** [comment]
Use them to write tests for bosh.

**Comment by gkiar** [comment]
@glatard -We have done this I believe?

**Comment by glatard** [comment]
I don't think so: https://github.com/boutiques/boutiques/tree/master/examples is still a mess.

**Comment by mathdugre** [comment]
To update the examples, does it only requires to change the `schema-version` in the descriptor? or does it require to import them from 0.4 or maybe something else?

**Comment by glatard** [comment]
Importing from version 0.4 requires running the importer indeed, as there are non-backward-compatible schema updates. Importing from prior to 0.4 requires manual editing.

**Comment by mathdugre** [comment]
Some of the descriptors in `boutiques/examples` fail to be imported to schema-version 0.5.

For example
-
when running
`bosh import 0.4 examples/configuration-file/config2.json examples/configuration-file/config.json`

I get this error:
`[ ERROR ] Additional properties are not allowed ('type' was unexpected)`
`Failed validating 'additionalProperties' in schema['properties']['output-files']['items']:`

Should I rewrite the failing part in the descriptor?

**Comment by glatard** [comment]
yes, the descriptor should be updated.

**Comment by glatard** [comment]
Let's also make sure that examples are properly integrated with the website, the doc, the README and/or the tutorial so that people can easily find and refer to them.

**Comment by glatard** [comment]
I think we want two kinds of examples:
1. Complete, published tools, such as FSL bet: zenodo.1482743
2. Minimal functional example (to be created, check the tutorial)
3. A complete example with all features (example 1 cleaned up)

**Comment by glatard** [comment]
Let's create a page on the website for examples, and refer to it in the README and tutorial.
